Gaijinizer patch/Map041.txt

```diff
@@ -79,260 +81,268 @@ Remove mind corruption if(v[42] \>= 100) #Dargoth
+play...#Ron
 > END STRING
 
 > BEGIN STRING
 つまり……貴方がどんなに必死に脱出しようとしても
 
 最終的には彼女に対峙することになってしまう…。
 > CONTEXT: Map041/events/5/pages/1/52/Dialogue/yuuma
-In other words...no matter how desperately you
-try to escape. Eventually you will face her ....#MTLed
+In other words... No matter how hard you tried to
+escape, you would eventually have to face her...#Ron
 > END STRING
 
 > BEGIN STRING
 そんな出来試合が待っているとも知らずに、
 必死に生きようとする貴方を見て…
 少し彼女の『シナリオ』を変えてやろうと思ったんです。
 > CONTEXT: Map041/events/5/pages/1/56/Dialogue/yuuma
-Without knowing that such a successful game is
-waiting, Seeing you trying to live desperately
-... I wanted to change her 'script' a little.#MTLed
+Seeing you desperately trying to live without
+knowing what kind of odds you were up against...
+I thought I'd try changing her "script".#Ron
 > END STRING
 
 > BEGIN STRING
 私はあの、昔から生命をオモチャ同然に扱う彼女が
 
 少々……好きではなかったものですから。
 > CONTEXT: Map041/events/5/pages/1/60/Dialogue/yuuma
-I treat her as a toy from the old days
-A little...because I did not like it.#MTLed
+From long ago she's treated lives as though they
+were no more than her playthings, and I...
+I slowly grew to dislike that.#Ron
 > END STRING
 
 > BEGIN STRING
 まさか………
 
 最後に僕を助けるために、わざと力を温存したのか？
 > CONTEXT: Map041/events/5/pages/1/64/Dialogue/kazuya
-No way ......... Lastly, did you intentionally
-save your power to help me?#MTLed
+No way...
+
+So in the end there did you intentionally retain
+your power so you could save me?#Ron
 > END STRING
 
 > BEGIN STRING
 あの時も話しましたが…
 
 この身体は、過度の魔力放出には耐えられないのです。
 > CONTEXT: Map041/events/5/pages/1/68/Dialogue/yuuma
-I also talked about that time ...
-This body can not stand excessive magic release.#MTLed
+I mentioned this before but...
+
+This body cannot withstand an excessive discharge
+of magical power.#Ron
 > END STRING
 
 > BEGIN STRING
 彼女をビビらせて、なおかつ彼女が去るまで……
 
 この身体は維持しておかねばなりませんでした。
 > CONTEXT: Map041/events/5/pages/1/72/Dialogue/yuuma
-Let her beat and until she leaves ...
-This body had to be maintained.#MTLed
+I had to maintain that amount of energy so I could
+frighten and drive off Hectate.#Ron
 > END STRING
 
 > BEGIN STRING
 ですが、あの直前の通路では少し本気を出しましたよ？
 
 あわよくば貴方が諦めて、私のものになればと…
 > CONTEXT: Map041/events/5/pages/1/77/Dialogue/yuuma
-But, did you get a little serious in the aisle
-just before that? If you do well, you will give
-up and it will be mine ...#MTLed
+But I did put in a bit of effort when you faced me
+in that hallway just before. With a bit of luck, I
+would have been able to make you mine.#Ron
 > END STRING
 
 > BEGIN STRING
 ああ……♡あの時の貴方の抱き心地…♡
 
 もしよろしかったら、また私に抱かせてくださいませ♡
 > CONTEXT: Map041/events/5/pages/1/81/Dialogue/yuuma
-Oh ...... your Hug comfort of that time...♡
-If it's wrong, please let me have it again#MTLed
+Aaah... ♡ The feeling of holding you... ♡
+If you'd like, please let me hold you again. ♡#Ron
 > END STRING
 
 > BEGIN STRING
 ねぇ、アイア…\\|\\^
 > CONTEXT: Map041/events/5/pages/1/90/Dialogue/kazuya
-Hey, Ia ...\\|\\^#MTLed
+Hey, Lily...\\|\\^#Ron
```

Comment:
Why translate this as Lilly? I honestly have no clue, I'm just curious.

Reply:
Heh, I just realised what I'd done. I'd misread and assumed he'd meant to say her real name but fudged it. So I thought I'd use "Lilly" since that's phonetically similiar to "Lucy". But reading it again I realise he's about to say "Iron Maiden" before stopping himself.